### PR TITLE
docs: Added code of conduct doc in root

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

This file is checked by GitHub for compliance so including it and pointing to where ours is.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Please see [our code of conduct policy](http://docs.librenms.org/General/CODE_OF_CONDUCT/).
```
Comment (murrant) [inline CODE_OF_CONDUCT.md]: I don't think this page will be built

Comment (laf) [inline CODE_OF_CONDUCT.md]: Do you mean because of the change I did in mkdocs? If not, then it already exists.

Comment (laf) [inline CODE_OF_CONDUCT.md]: Or do you mean this new file? If it's the new file then it's part of GitHubs way to show compliance for open source projects.

**Comment by murrant** [comment]
I trust you :)

**Pull request merged**